main/guides/orchestration/getting-started/key-concepts.md

```diff
@@ -0,0 +1,235 @@
+    timer,
+  });
+  return holder;
+}
+```
+
+
+### Baggage
+
+baggage is a MapStore that provides a way to preserve the state and behavior of objects between [smart contract upgrades](https://docs.agoric.com/guides/zoe/contract-upgrade) in a way that preserves the identity of objects as seen from other [vats](#vat). In the provided contract, baggage is used to ensure that the state of various components is maintained even after the contract is upgraded.
+
+```javascript
+export const start = async (zcf, privateArgs, baggage) => {
+  ...
+  const { accountsStorageNode } = await provideAll(baggage, {
+    accountsStorageNode: () => E(storageNode).makeChildNode('accounts'),
+  });
+  ...
+}
+```
+
+
+### Exo
+An Exo object is an exposed Remotable object with methods (aka a [Far](/glossary/#zone) object), which is normally defined with an [InterfaceGuard](#todo-interface-guard) as a protective outer layer, providing the first layer of defensiveness.
+
+This [@endo/exo](https://github.com/endojs/endo/tree/master/packages/exo) package defines the APIs for making Exo objects, and for defining ExoClasses and ExoClassKits for making Exo objects.
+
+```javascript
+const publicFacet = zone.exo(
+  'StakeAtom',
+  M.interface('StakeAtomI', {
+    makeAccount: M.callWhen().returns(M.remotable('ChainAccount')),
+    makeAccountInvitationMaker: M.callWhen().returns(InvitationShape),
+  }),
+  {
+    async makeAccount() {
+      trace('makeAccount');
+      const holder = await makeAccountKit();
+      return holder;
+    },
+    makeAccountInvitationMaker() {
+      trace('makeCreateAccountInvitation');
+      return zcf.makeInvitation(
+        async seat => {
+          seat.exit();
+          const holder = await makeAccountKit();
+          return holder.asContinuingOffer();
+        },
+        'wantStakingAccount',
+      );
+    },
+  },
+);
+```
+
+
+
+### Zones
+Each [Zone](/glossary/#zone) provides an API that allows the allocation of [Exo objects](#exo) and Stores [(object collections)](https://github.com/Agoric/agoric-sdk/tree/master/packages/store/README.md) which use the same underlying persistence mechanism. This allows library code to be agnostic to whether its objects are backed purely by the JS heap (ephemeral), pageable out to disk (virtual), or can be revived after a vat upgrade (durable).
+
+
+See [SwingSet vat upgrade documentation](https://github.com/Agoric/agoric-sdk/tree/master/packages/SwingSet/docs/vat-upgrade.md) for more example use of the zone API.
+
+```javascript
+const zone = makeDurableZone(baggage);
+...
+zone.subZone('vows')
+```
+
+
+### ChainHub
+
+The `makeChainHub` utility manages the connections and metadata for various blockchain networks. It simplifies accessing and interacting with multiple chains, providing a unified interface for the orchestration logic to manage cross-chain operations effectively.
+
+```javascript
+const chainHub = makeChainHub(remotePowers.agoricNames);
+```
+
+# Recorder Kit Makers
```

Comment:
**technical**: Recorders are not an orchestration concept. They're for vstorage.